h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java

```diff
@@ -568,5 +568,19 @@ public void testReadComplexFile() throws Exception {
         stat.execute("DROP TABLE IF EXISTS TABLE_POINTS_READ");
         stat.close();
     }
+
+    @Test
+    public void testReadProperties() throws Exception {
+        Statement stat = connection.createStatement();
+        stat.execute("DROP TABLE IF EXISTS TABLE_PROPERTIES_READ;");
+        stat.execute("CALL GeoJsonRead("+ StringUtils.quoteStringSQL(GeojsonImportExportTest.class.getResource("data.geojson").getPath()) + ", 'TABLE_PROPERTIES_READ');");
```

Comment:
It is not useful to use a such big file (`data.geojson`) if you test only the very first line. Maybe a smaller file will be better.